src/components/shared/Input/Input.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Input.scss';
+
+const Input = ({ setValueToState, label, ...otherProps }) => {
+  return (
+    <div className='container'>
+      <input className='container__input' {...otherProps} />
+      {label ? (
+        <label
+          className={`${
+            otherProps.value.length ? 'container__shrink' : ''
+          }  container__label`}
+        >
+          {label}
+        </label>
```

Comment:
tutaj fajnie nadajesz dynamicznie klase to samo powinno byc w buttonie.

Reply:
w battonie w jakim celu ?

Reply:
w buttonie nadawalbym w propsie jakas nazwe typu "accept" albo "exit" i na bazie tego propsa nadawal dynamicznie jakas klase czyli zmienialbym kolor cos w tym stylu mysle

Reply:
ok rozumiem,  zrobi się